deploy/docker/Dockerfile

```diff
@@ -49,7 +49,16 @@ USER root
 # Run dependencies
 RUN runDeps="sudo libsnappy-dev" \
  && apt-get update \
- && apt-get install -y --no-install-recommends $runDeps
+ && apt-get install -y --no-install-recommends $runDeps \
+ && apt-get install -y gnupg \
+ && apt-get install -y curl \
+ && apt-get install -y jq \
+ && apt-get install -y wget
+
+ # Install kubectl
+ADD https://storage.googleapis.com/kubernetes-release/release/v1.6.12/bin/linux/amd64/kubectl /usr/local/bin/kubectl
```

Comment:
Is this version too low?

Reply:
updated to 1.15.0 according to their [doc page](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux)